src/context/AppProvider/AppContext.tsx

```diff
@@ -322,7 +325,7 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     const fetchForexRate = marketApi.endpoints.findByFiatSymbol.initiate
     dispatch(getFiatPriceHistory({ symbol, timeframe }))
     dispatch(fetchForexRate({ symbol }))
-  }, [dispatch, selectedCurrency])
+  }, [DEFAULT_HISTORY_TIMEFRAME, dispatch, selectedCurrency])
```

Comment:
revert

Reply:
Reverted in https://github.com/shapeshift/web/pull/2209/commits/b1ad1a4fe1bbff1d36bb0d4501807fadb5d23b6b